envFiles/.env.production

```
@@ -106,8 +106,15 @@ FRONTEND_URL="http://localhost:4321"
 
 
 ##### OTEL and Observability #####
-API_OTEL_ENABLED=true
+API_OTEL_ENABLED=false
 API_OTEL_SERVICE_NAME="talawa-api"
 API_OTEL_ENVIRONMENT="production"
 API_OTEL_SAMPLING_RATIO="1"
-API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces"#TODO: after integrating Observability tool
\ No newline at end of file
+API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces" #TODO: Ensure this is the correct endpoint for your OTLP exportervability tool
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_OTEL_EXPORTER_OTLP_ENDPOINT key should go before the API_OTEL_SAMPLING_RATIO key

(UnorderedKey)